unity/Assets/Maroon/reusableGui/Menu/PrefabsColumns/scrMenuColumnAudio.cs

```diff
@@ -20,18 +20,20 @@ public class scrMenuColumnAudio : MonoBehaviour
 
     void Start()
     {
-        // Listen to music slider
+        // Apply initial slider value from Music AudioSource and listen to music slider
         if (_musicSlider)
         {
-            _musicSlider.GetComponent<Slider>().onValueChanged
-                .AddListener((value) => this.OnChangeMusicSlider(value));
+            var musicSliderComponent = _musicSlider.GetComponent<Slider>();
+            musicSliderComponent.value = SoundManager.Instance.MusicVolume;
+            musicSliderComponent.onValueChanged.AddListener(this.OnChangeMusicSlider);
```

Comment:
Music: Set initial slider value to its AudioSource's volume before adding the listener as before.